### PR TITLE
Alliance update + Scaleout Project

### PR DESCRIPTION
**Pull request by morganekmefjord**
Updated the helm charts to facilitate changes in alliance setup.

Added a new helm chart for scaleout project with project level minio.

### CODE REVIEW

File: scaleout/alliance/templates/orchestrator-deployment.yaml [hunk 1]
```diff
@@ -25,31 +25,74 @@ spec:
...
-        resources: {}
-      #  volumeMounts:
-      #  - mountPath: /app/
-      #    name: orchestrator-claim0
-      #  - mountPath: /app/data
-      #    name: orchestrator-claim1
+        volumeMounts:
+        - name: git
+          mountPath: /app/orchestrator
+          subPath: orchestrator/federatedaveraging
```
Comment (ahellander) [inline scaleout/alliance/templates/orchestrator-deployment.yaml]: Moving forward, there will be more types of orchestrators than federatedaveraging. Should the filname of the chart reflect this already now? (federatedaveraging-deployment.yaml)

File: scaleout/project/values.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+
+
+
+
+labs:
+  image: jupyter/minimal-notebook:latest
+  repository: "git@github.com:scaleoutsystems/scaleout-alliance-orchestrator-sdk.git"
```
Comment (ahellander) [inline scaleout/project/values.yaml]: scaleout-cli.git ?

File: scaleout/alliance/templates/orchestrator-deployment.yaml (hunk 1 above)
Comment (morganekmefjord) [reply scaleout/alliance/templates/orchestrator-deployment.yaml]: We will reuse the same template and just replace that with a custom values overridable subdirectory override.

File: scaleout/project/values.yaml (hunk 2 above)
Comment (morganekmefjord) [reply scaleout/project/values.yaml]: No this is testing to load a private repository with a custom key present. Much similar to the intended tennant/project instance (and key). Hence the orchestrator. (perhaps a bad example but working).

**Pull request merged**